packages/global/config/identity-x.js

```diff
@@ -0,0 +1,42 @@
+const IdentityXConfiguration = require('@parameter1/base-cms-marko-web-identity-x/config');
+const newrelic = require('newrelic');
+
+module.exports = ({
+  appId,
+  hiddenFields = [
+    'street',
+    'addressExtra',
+    'organization',
+    'organizationTitle',
+    'phoneNumber',
+  ],
+  defaultCountryCode = '',
+  requiredServerFields = [
+    'givenName',
+    'familyName',
+    'city',
+    'regionCode',
+    'countryCode',
+  ],
+  requiredClientFields = [
+    'givenName',
+    'familyName',
+    'city',
+    'regionCode',
+    'countryCode',
+  ],
```

Comment:
Since they haven't selected them, the custom values for all of these configs should be removed to use the IdentityX defaults.